app/controllers/api/v1/va_notify_controller.rb

```diff
@@ -7,40 +7,60 @@ class Api::V1::VaNotifyController < Api::ApplicationController
   #
   # Response: Update corresponding Notification status
   def notifications_update
-    send "#{required_params[:notification_type]}_update"
+    send "notification_update"
```

Comment:
```suggestion
    send_sqs_message
    render json: { message: "Email Notification successfully updated: ID #{required_params[:id]}" }
```

What do you think about getting rid of the `notification_update` method and leaving this one?